tests/backend/routes/test_signup.py

```
@@ -126,3 +121,178 @@ def boom(admin_email, notification):
+    _capture_user_email(monkeypatch)
+    _stub_store(monkeypatch)
+
+    request_id, _ = _pending_request(tmp_path)
+    resp = test_client.post(f"/signup/approve?id={request_id}&token=not-the-token")
+    assert resp.status_code == 400
+    # Nothing was provisioned.
+    assert not (tmp_path / "accounts" / "jane").exists()
+
+
+def test_approve_unknown_request_rejected(client, monkeypatch):
+    test_client, tmp_path = client
```

Ruff (0.15.17)

[warning] Unpacked variable `tmp_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)